<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="Armenian" country="Armenia"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>单词</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_word">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>词性</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>61</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>n</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>v</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pron</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>adj</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>adv</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>prep</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pron</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>conj</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>num</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>int</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>释义</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_mean">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>75</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>例句</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>英</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_enges">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_chnes">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
